zproject/computed_settings.py

```diff
@@ -280,6 +280,7 @@
         "OPTIONS": {
             "connection_factory": TimeTrackingConnection,
             "cursor_factory": TimeTrackingCursor,
+            "connect_timeout": 2,  # Minimum supported non-zero value
```

Comment:
Probably worth putting a block comment here documenting why we're using the minimum supported value; just pulling some content from the commit message should do.